web/src/features/modals/LayersModal.tsx

```diff
@@ -0,0 +1,141 @@
+import GlassContainer from 'components/GlassContainer';
+import SwitchToggle from 'components/ToggleSwitch';
+import { weatherButtonMap } from 'features/map-controls/MapControls';
+import { useDarkMode } from 'hooks/theme';
+import { useAtom, useAtomValue } from 'jotai';
+import { XIcon } from 'lucide-react';
```

Comment:
I this is an alias of the same icon but in other places we use this:

```suggestion
import { X } from 'lucide-react';
```

Reply:
Yeah, I'd like us to switch to *Icon everywhere, it's nice and clear!

Reply:
Will create a follow up PR to do this!